src/EFCore.MySql/Infrastructure/CharSetConfig.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Pomelo Foundation. All rights reserved.
+// Licensed under the MIT. See LICENSE in the project root for license information.
+
+using Pomelo.EntityFrameworkCore.MySql.Storage.Internal;
+
+namespace Pomelo.EntityFrameworkCore.MySql.Infrastructure
+{
+    public class CharSetConfig
+    {
+        public CharSetInfo ServerDefaultCharSetInfo { get; private set; }
```

Comment:
Since we only need the bytes per character I think it would be less confusing to the user for this to be an `int`